docs/output.md

```diff
@@ -14,457 +14,415 @@ All paths are relative to the top-level results directory.
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
 - [Preprocessing](#preprocessing)
+  - [Prepare input files](#prepare-input)
+    - [Trim adapters](#trimming)
+    - [Split fastq files](#split)
   - [Map to Reference](#map-to-reference)
     - [BWA](#bwa)
     - [BWA-mem2](#bwa-mem2)
-  - [Mark Duplicates](#mark-duplicates)
-    - [GATK MarkDuplicates](#gatk-markduplicates)
-    - [GATK MarkDuplicates Spark](#gatk-markduplicates-spark)
-  - [Base (Quality Score) Recalibration](#base-quality-score-recalibration)
+    - [DragMap](#dragmap)
+  - [Duplicate Marking](#duplicate-marking)
+    - [GATK MarkDuplicates (Spark)](#gatk-markduplicates)
+  - [Base Quality Score Recalibration](#base-quality-score-recalibration)
```

Comment:
The section title in the text below is still "Base (Quality Score) Recalibration". Do we aim for the titles in the content list to match the actual section titles below?  I didn't check the other titles. (I'm too lazy).